src/components/Users/UsersContainer.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+
+import Users from './Users';
+import { followActionCreator, unfollowActionCreator, setUsersActionCreator } from '../../redux/reducers/usersReducer';
```

Comment:
This belongs to the "common to project" group of import.
Why? Because it's outside of the current "Users" folder.
I already asked your to reorganize the order of imports several times.

Reply:
/fxd